src/ConfigManager.h

```diff
@@ -0,0 +1,172 @@
+    static G4String      GetParticleType()                  { return Instance()->particleType_; }
+    static G4bool        GetDecayAtTimeZero()               { return Instance()->decayAtTimeZero_; }
+    static G4bool        GetIsotropic()                     { return Instance()->isotropic_; }
+    static G4bool        GetOverrideVertexPosition()        { return Instance()->overrideVertexPosition_; }
+    static G4bool        GetPrintParticleInfo()             { return Instance()->printParticleInfo_; }
+    static G4String      GetInputFile()                     { return Instance()->inputFile_; }
+    static G4String      GetOutputFile()                    { return Instance()->outputFile_; }
+    static G4String      GetMarleyJson()                    { return Instance()->marleyJson_; }
+    static G4String      GetGenerator()                     { return Instance()->generator_; }
+    static G4String      GetGenieFormat()                   { return Instance()->genieFormat_; }
+    static G4bool        GetMultirun()                      { return Instance()->multirun_; }
+    static G4ThreeVector GetMomentumDirection()             { return Instance()->momentumDirection_; }
+    static G4double      GetVertexX()                       { return Instance()->vertexX_; }
+    static G4double      GetVertexY()                       { return Instance()->vertexY_; }
+    static G4double      GetVertexZ()                       { return Instance()->vertexZ_; }
+    static G4int         GetNAr39Decays()                   { return Instance()->nAr39Decays_; }
+    static G4int         GetNAr42Decays()                   { return Instance()->nAr42Decays_; }
+    static G4int         GetNKr85Decays()                   { return Instance()->nKr85Decays_; }
+    static G4int         GetNCo60Decays()                   { return Instance()->nCo60Decays_; }
+    static G4int         GetNK40Decays()                    { return Instance()->nK40Decays_; }
+    static G4int         GetNK42Decays()                    { return Instance()->nK42Decays_; }
+    static G4int         GetNBi214Decays()                  { return Instance()->nBi214Decays_; }
+    static G4int         GetNPb214Decays()                  { return Instance()->nPb214Decays_; }
+    static G4int         GetNPo210Decays()                  { return Instance()->nPo210Decays_; }
+    static G4int         GetNRn222Decays()                  { return Instance()->nRn222Decays_; }
+    static G4double      GetEventCutoff()                   { return Instance()->eventCutoff_; }
+    static G4double      GetEventWindow()                   { return Instance()->eventWindow_; }
+    static G4bool        GetSNTimingOn()                    { return Instance()->snTimingOn_; }
+    static G4String      GetTh2Name()                       { return Instance()->th2Name_; }
+    static G4bool        GetDetectorConfiguration()         { return Instance()->detectorConfiguration_; }
```

Comment:
This needs a more understandable name if it's just returning a bool.  Maybe `UseHDDetector()`?

Reply:
fixing in bugfix/v3.0.0_revisions branch